acceptance/features/release.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Golden Container Image
+
+    Scenario: Red Hat collection
+        Given there is a sample golden-container policy input
+        And there is a policy config
```

Comment:
This makes it easier to read in my mind. And allows the `"golden-container"` to be a parameter.

```suggestion
        Given sample policy input "golden-container"
          And a policy config:
```

Reply:
https://github.com/enterprise-contract/ec-policies/pull/866